File: TDDMicroExercises.ProposedSolution/CSharp/TirePressureMonitoringSystem.SomeDependencies/AnAlarmClient2.cs
using System;
namespace TDDMicroExercises.OneSolution.TirePressureMonitoringSystem.SomeDependencies
{
    public class AnAlarmClient2
    {
		// A class with the only goal of simulating a dependency on Alart
		// that has impact on the refactoring.
		
        private void DoSomething()
        {
            Alarm anAlarm = new Alarm();
            anAlarm.Check();
            bool isAlarmOn = anAlarm.AlarmOn;
        }
    }
}

File: TDDMicroExercises/Java/TirePressureMonitoringSystem/src/main/java/tddmicroexercises/tirepressuremonitoringsystem/somedependencies/AnAlarmClient2.java
package tddmicroexercises.tirepressuremonitoringsystem.somedependencies;

import tddmicroexercises.tirepressuremonitoringsystem.Alarm;

public class AnAlarmClient2
{
    // A class with the only goal of simulating a dependency on Alert
    // that has impact on the refactoring.

    private void DoSomething()
    {
        Alarm anAlarm = new Alarm();
        anAlarm.check();
        boolean isAlarmOn = anAlarm.isAlarmOn();
    }
}
